autoscheduler/frontend/src/package.json

```diff
@@ -49,6 +49,8 @@
     "prop-types": "^15.7.2",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
+    "react-google-button": "^0.5.3",
```

Comment:
t looks like you didn't add `typefaces-roboto` to the package.json, but the buttons have the Roboto font. Did you load it using a CDN? Or does it just work because I happen to have that font cached on my system?